### PR TITLE
[ISSUE-361] Update CI with operator

### PR DESCRIPTION
**Pull request by safronovD**
### Issue #361 

Proposal to integrate csi-baremetal-operator with CI procedures using in development at the moment

## PR checklist
- [x] Add link to the issue
- [x] Choose PR label
- [ ] New unit tests added
- [ ] Modified code has meaningful comments
- [ ] All TODOs are linked with the issues
- [ ] All comments are resolved

## Testing
_Link to custom CI build_

### CODE REVIEW

File: docs/proposals/operator-ci-procedures.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+- csi-baremetal-scheduler-extender-patcher docker image
+
+- csi-baremetal-##mgr docker image
+
+The artifacts have equal version gotten from 'make version'
+
+#### csi-baremetal-operator-ci
+
+e2e testing stage: deploy csi-baremetal-CR with operator + perform base tests
```
Comment (AnastasiaShemyakinskaya) [inline docs/proposals/operator-ci-procedures.md:48]: What do you mean by `base tests`?

File: docs/proposals/operator-ci-procedures.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+- csi-baremetal-node-controller docker image -> green
+
+- csi-baremetal-scheduler-extender docker image -> green
+
+- csi-baremetal-scheduler-extender-patcher docker image -> green
+
+- csi-baremetal-##mgr docker image -> green
+
+#### csi-baremetal-operator-ci
```
Comment (AnastasiaShemyakinskaya) [inline docs/proposals/operator-ci-procedures.md:71]: Can you clarify why there are two separate `csi-baremetal-operator-ci` with different tests and versions changes?

File: docs/proposals/operator-ci-procedures.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+4. Update csi-baremetal-build, csi-baremetal-ci, csi-custom-build/ci jobs
+
+5. Update other affected jobs
+
+## Open issues
+
+ID | Name | Descriptions | Status | Comments
+---| -----| -------------| ------ | --------
+ISSUE-1 | csi-baremetal-scheduler-ci  |  ci for https://github.com/dell/csi-baremetal-scheduling may be included to this proposal | Open  | 
+ISSUE-2 | csi-baremetal-node-controller renaming  |  Can we will have conflicts in artifactory until renaming? | Open  | It will be solved if csi-baremetal-operator is tagged with specific version
```
Comment (AnastasiaShemyakinskaya) [inline docs/proposals/operator-ci-procedures.md:127]: Can you add more information about that issue? It's not really clear to me

File: docs/proposals/operator-ci-procedures.md [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+
+- csi-baremetal-operator version (empty -> build)
+
+- csi-baremetal branch (default master)
+
+- csi-baremetal-operator branch (default master)
+
+## Implementation
+
+1. Create and test new csi-baremetal-operator-build job
```
Comment (mishoyama) [inline docs/proposals/operator-ci-procedures.md:112]: Not sure this is a good idea to mention internal jobs here.

**Pull request closed without merging**